static/app/components/replays/breadcrumbs/utils.tsx

```diff
@@ -1,14 +1,20 @@
+// import {isBreadcrumbTypeDefault} from 'sentry/types/breadcrumbs';
 import {BreadcrumbType, Crumb} from 'sentry/types/breadcrumbs';
 
 /**
  * Generate breadcrumb descriptions based on type
  */
-export function getDescription(crumb: Crumb) {
+export function getDescription(crumb: Crumb, startTimestampMs: number) {
   switch (crumb.type) {
     case BreadcrumbType.NAVIGATION:
       return `${crumb.data?.to ?? ''}`;
     case BreadcrumbType.DEFAULT:
       return JSON.stringify(crumb.data);
+    case BreadcrumbType.DEBUG:
+      if (crumb.data?.action === 'largest-contentful-paint' && crumb.timestamp) {
```

Comment:
I would pull this out of this switch block and create its own based on `crumb.data.action` as it's more specific than `crumb.type`. I would guess we'll have more of these based on `action`.